wgpu/src/shader/quad.wgsl

```diff
@@ -5,6 +5,14 @@ struct Globals {
 
 @group(0) @binding(0) var<uniform> globals: Globals;
 
+fn premultiply(color: vec4<f32>) -> vec4<f32> {
+    return vec4(color.xyz * color.a, color.a);
+}
+
+fn blend(over: vec4<f32>, under: vec4<f32>, alpha: f32) -> vec4<f32> {
+    return over * alpha + under * (1.0 - alpha);
```

Comment:
same here, could use `mix(under, over, alpha)`

Reply:
Or maybe just get rid of this `blend` fn entirely and use `mix` instead

Reply:
I see. Alpha blending is equivalent to mixing with premultiplied alphas.